src/python/pants/backend/codegen/protobuf/python/additional_fields.py

```diff
@@ -3,11 +3,25 @@
 
 from pants.backend.codegen.protobuf.target_types import ProtobufLibrary
 from pants.backend.python.target_types import PythonInterpreterCompatibility
+from pants.engine.target import StringField
 
 
 class ProtobufPythonInterpreterCompatibility(PythonInterpreterCompatibility):
     alias = "python_compatibility"
 
 
+class PythonSourceRootField(StringField):
+    """The source root to generate python sources under.
+
+    If unspecified, the source root the protobuf_library is under will be used.
+    """
+
+    alias = "python_source_root"
+    default = None
```

Comment:
That's the `default` already, so this is redundant. Fine to keep if you wanted it for extra clarity.